src/test/run-pass-fulldeps/qquote.rs

```diff
@@ -9,59 +9,56 @@
 // except according to those terms.
 
 // ignore-cross-compile
-// ignore-pretty
 
 #![feature(quote, rustc_private)]
 
 extern crate syntax;
 
-use syntax::ast;
-use syntax::codemap;
-use syntax::parse;
-use syntax::print::pprust;
+use syntax::codemap::DUMMY_SP;
+use syntax::print::pprust::*;
 
-trait FakeExtCtxt {
```

Comment:
I rewrote this test independently from your branch (from when it was ignore-test-ed and nonfunctional) and when rebasing I figured that my test already covered everything yours does, so I just took mine. (Also, it seems kind of fragile to me IMO to fake the `ExtCtxt` methods that quasiquote happens to need, instead of creating an actual `ExtCtxt`.)